app/webpack.config.js

```diff
@@ -217,6 +217,8 @@ module.exports = {
                     '**/*.stories.jsx',
                     // skip MD documentation
                     'src/docs/docsEditingGuide.md',
+                    // skip sketch files
+                    '**/*.sketch',
```

Comment:
sketch files?

Reply:
Commit message for 61404a8d234f66a8c78d6c9aff297a7a4987a27e:
> **Let .sketch files be unused**
> I tend to keep a local sketch file for components I work with (slices, simplified svgs, etc). Don’t want them to be catched by UnusedFilesWebpackPlugin.

😃

Reply:
👍